sensor_type: 1
sensor_direction: 1
sensor_orientation: [0.0, -10.0, 0.0]